workflows/scripts/aggregate_cells.py

```diff
@@ -146,16 +140,27 @@ def aggregate(
     metadata_dir_path = str(snakemake.input["metadata"])
     cell_count_output = str(snakemake.output["cell_counts"])
     aggregate_output = str(snakemake.output["aggregate_profile"])
-    config_path = str(snakemake.params["aggregate_config"])
+    single_cells_config = snakemake.params["single_cell_config"]["params"]
+    aggregate_config = snakemake.params["aggregate_config"]["params"]
     log_path = str(snakemake.log)
 
+    # print(plate_data)
+    # print(barcode_path)
+    # print(metadata_dir_path)
+    # print(cell_count_output)
+    # print(aggregate_output)
+    # print(configs)
+    # print("#################3")
+    # print(log_path)
```

Comment:
probably to delete

Reply:
I totally forgot to remove this. This is an example of my furstration.